sunpy/util/sysinfo.py

```diff
@@ -118,7 +120,7 @@ def system_info():
         print('{} : {}'.format(sys_info, sys_prop[sys_info]))
 
     if sys_prop['System'] == "Linux":
-        distro = " ".join(platform.linux_distribution())
+        distro = " ".join(linux_distribution())
```

Comment:
Is it really worth ~1500 extra lines of code just to make it possible to distinguish between linux distros? How often has this information actually come in handy?

Reply:
I don't think it's bad idea in the long run. We should be looking to expand how smart and what detailed info we can get from our sysinfo and  think this is part 1 of that.

Reply:
Or if you prefer we can just add it as a dependency  like we are doing for `importlib_metadata`